README.md

```diff
@@ -366,6 +367,27 @@ AWS_SECRET_ACCESS_KEY=
 CB_AWS_ACCOUNT_ID=
 ```
 
+### Running the Consumption service in IDEA
+
+After importing the `cloudbreak` repo root, launch the Consumption application by executing the `com.sequenceiq.consumption.ConsumptionApplication` class (set `Use classpath of module` to `cloudbreak.cloud-consumption.main`) with the following JVM options:
```

Comment:
can you run the application from cbd as well?

Reply:
you mean by adding it to the CB_LOCAL_DEV_LIST?

Reply:
yes

Reply:
yes if you remove it the consumption container will  run by cbd command.